Heading/Heading.js

```diff
@@ -110,17 +108,27 @@ const HeadingBase = kind({
  * @public
  */
 const HeadingDecorator = compose(
-	setPropTypes({
-		marqueeOn: PropTypes.oneOf(['hover', 'render'])
-	}),
-	defaultProps({
-		marqueeOn: 'render'
-	}),
 	Pure,
 	MarqueeDecorator({invalidateProps: ['remeasure', 'slotSize']}),
 	Skinnable
 );
 
+HeadingDecorator.PropTypes = /** @lends sandstone/Heading.HeadingDecorator.prototype */ {
+	/**
+	 * Determines what triggers the header content to start its animation. Valid values are
+	 * `'hover'` and `'render'`. The default is `'render'`.
+	 *
+	 * @type {String}
+	 * @default 'render'
```

Comment:
Is there any reason the type is different with ui/MarqueeDecorator?
```suggestion
	 * Determines what triggers the header content to start its animation.
	 *
	 * @type {('focus'|'hover'|'render')}
	 * @default 'render'
```

Reply:
No reason :) I just copied it from another place. I'll replace it.